specfile/constants.py

```diff
@@ -244,3 +244,5 @@
     "x86_64_v4",
     "xtensa",
 }
+
+FILE_RW = {"mode": "r+", "encoding": "utf8", "errors": "surrogateescape"}
```

Comment:
Sorry, I meant the `encoding` and `errors` arguments that are shared among all `encode()`/`decode()`/`open()` methods. I would also make it a `Specfile`-specific constant, or at least move it to `specfile.py`, this file is intended to hold RPM constants and is supposed to be updated programatically. I would probably call the constant something like `ENCODING`, the `errors` argument is tightly tied to encoding.